src/pip/_internal/commands/uninstall.py

```diff
@@ -108,6 +108,8 @@ def run(self, options: Values, args: List[str]) -> int:
             )
             if uninstall_pathset:
                 uninstall_pathset.commit()
+                if req.files_skipped:
+                    return ERROR
```

Comment:
Hmm, is it correct to break early for anything skipped here? Maybe the loop should collect `files_skipped` status from each `req` and return `ERROR` after the loop instead?

Reply:
A test that uninstalls multiple packages will be needed too.

Reply:
Fixed.